bin/logstash.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1beta1
+kind: Config
+id: logstash-config
+items:
+  - apiVersion: v1beta1
+    kind: ReplicationController
+    id: logstash-controller
+    desiredState:
+      replicas: 1
+      replicaSelector:
+        component: logstash-viewer
+      podTemplate:
+        desiredState:
+          manifest:
+            version: v1beta1
+            id: logstash-server
+            containers:
+              -
+                name: logstash-image
+                image: fabric8/logstash:latest
```

Comment:
Where is this image? It's not on Docker hub - we need to get it there before we accept this PR.